### PR TITLE
Update kolkata to add information and images

### PR DESCRIPTION
**Pull request by nicholas-owen**
some text and image updates

### CODE REVIEW

File: web_source/cities/asia/india/kolkata.md [hunk 1]
```diff
@@ -4,11 +4,22 @@ Country: India (Asia)
...
 ### What we would like to see there
 
-FIXME
+#### City of Palaces, Kolkata
+![City of Palaces](https://encrypted-tbn3.gstatic.com/images?q=tbn:ANd9GcSRV5I_dnjtLFhIAST8Ie1PQMpmocJtM5xR8M-sZRqwUAq5l13VwwYygxADVQVu)
+
+#### Calcutta, Kolkata
+![](https://encrypted-tbn2.gstatic.com/images?q=tbn:ANd9GcQiekKPB9PwMURmo_fZZ0WPizg7qxH9GNMg3Y6qQyOQhfqSx1YMYUAqCMly8Zk9)
 
 ### Extra information
```
Comment (BenSmithUCL) [inline web_source/cities/asia/india/kolkata.md:24]: Could you add a link to the wikipedia page please?

**Review by BenSmithUCL — CHANGES_REQUESTED** [review]
Looks great but you need to update the wiki link!

File: web_source/cities/asia/india/kolkata.md (hunk 1 above)
Comment (nicholas-owen) [reply web_source/cities/asia/india/kolkata.md:24]: Of course :D

**Review by BenSmithUCL — APPROVED** [review]
Thanks for the update!

**Pull request merged**